manifests/gunicorn.pp

```diff
@@ -66,18 +66,21 @@
 # Marc Fournier
 #
 define python::gunicorn (
-  $ensure        = present,
-  $virtualenv    = false,
-  $mode          = 'wsgi',
-  $dir           = false,
-  $bind          = false,
-  $environment   = false,
-  $owner         = 'www-data',
-  $group         = 'www-data',
-  $appmodule     = 'app:app',
-  $osenv         = false,
-  $timeout       = 30,
-  $template      = 'python/gunicorn.erb',
+  $ensure            = present,
+  $virtualenv        = false,
+  $mode              = 'wsgi',
+  $dir               = false,
+  $bind              = false,
+  $environment       = false,
+  $owner             = 'www-data',
+  $group             = 'www-data',
+  $appmodule         = 'app:app',
+  $osenv             = false,
+  $timeout           = 30,
+  $access_log_format = false,
```

Comment:
These three lines are the only additions. The rest are whitespace alignment from the long variable name $access_log_format.